tests/e2e/README.md

```diff
@@ -0,0 +1,44 @@
+## Prerequisites
+
+1.  You have a CloudFormation stack called `SdkReleaseV3IntegTestResourcesStack` in the AWS account you used to run the
+    integration test. If not you can deploy the stack with:
+
+    ```console
+    aws cloudformation deploy --stack-name SdkReleaseV3IntegTestResourcesStack --template-file ./IntegTestResourcesStack.template.json --capabilities CAPABILITY_IAM
+    ```
+
+1.  You have `ubuntu/standard/4.0` docker image built. If not, please follow [the guide to build the image](https://github.com/aws/aws-codebuild-docker-images#how-to-build-docker-images).
```

Comment:
Installation of `aws/codebuild/standard:4.0` fails on AL2

Created GitHub issue at https://github.com/aws/aws-codebuild-docker-images/issues/375

Reply:
The issue was fixed by an alternative provided in https://github.com/aws/aws-codebuild-docker-images/issues/375#issuecomment-668946363